apps/angular/projection/src/app/ui/list-item/list-item.component.ts

```diff
@@ -17,19 +14,5 @@ import { CardType } from '../../model/card.model';
 })
 export class ListItemComponent {
   @Input() id!: number;
```

Comment:
don't  actually need the ID, it can be accessed from the parent cards

Reply:
ah yeah! 😀